pkg/route/registry/route/etcd/etcd.go

```diff
@@ -53,11 +52,19 @@ type StatusREST struct {
 	store *registry.Store
 }
 
+// StatusREST implements Patcher
+var _ = kapirest.Patcher(&StatusREST{})
+
 // New creates a new route resource
 func (r *StatusREST) New() runtime.Object {
 	return &api.Route{}
 }
 
+// Get retrieves the object from the storage. It is required to support Patch.
```

Comment:
All updates will be done with patch, not sure starting when, but we should be prepared for that.